src/main/java/me/damonkelley/tictactoe/Players.java

```diff
@@ -0,0 +1,19 @@
+package me.damonkelley.tictactoe;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Players {
+    private final List<Player> players;
+
+    public Players(Player playerA, Player playerB) {
+        players = Arrays.asList(playerA, playerB);
+    }
+
+    public Player get(Marker x) {
```

Comment:
I think using `x` as your variable name here is unclear. Is the stream always filtering for the player with the `X` marker? I would prefer a more clear variable name here.